spec/features/admin/admin_users_spec.rb

```diff
@@ -33,19 +33,19 @@
     it "should apply defaults to user" do
       click_button "Create user"
       user = User.find_by(username: 'bang')
-      user.projects_limit.should == Gitlab.config.gitlab.default_projects_limit
-      user.can_create_group.should == Gitlab.config.gitlab.default_can_create_group
+      expect(user.projects_limit).to eq(Gitlab.config.gitlab.default_projects_limit)
+      expect(user.can_create_group).to eq(Gitlab.config.gitlab.default_can_create_group)
     end
 
     it "should create user with valid data" do
       click_button "Create user"
       user = User.find_by(username: 'bang')
-      user.name.should ==  "Big Bang"
-      user.email.should == "bigbang@mail.com"
+      expect(user.name).to eq("Big Bang")
+      expect(user.email).to eq("bigbang@mail.com")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.